hokusai/lib/default_branch.py

```diff
@@ -0,0 +1,9 @@
+from hokusai.lib.common import shout
+
+def detect_branch():
+  try: 
+    default_branch = shout("git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'")
```

Comment:
This git command will look up the local default HEAD, not the remote one. It seems like hokusai is always fetching remote before running automations and should already have the current version of the default branch also being mirrored locally. There is however another command with `git remote show` that could even make another call to the actual git remote in order to check live what is the current status of HEAD there. Under the circumstances it seems redundant though... Thanks @sonecabr for explaining this.

Reply:
Looking through `shout` and without any documentation there, it's not clear to me what it does and why. @kajatiger another way to execute such commands would be 
```
import os
os.system("git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'")
```
but I strongly suspect `shout()` is the desired way, so all good!

In a local python console in another repo I get the following output:
```
>>> import os
>>> os.system("git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'")
master
0
>>>
```
Guess that `0` is not desired so I think you're doing the right thing here!

Reply:
We are already using `shout` in combination with running a git command in another place in hokusai, so this is where we took this from. I also ran the plain git command in 2 different projects one with "main" and one with "master" to check if it actually works as expected and it should be fine.